src/main/java/UpdateOpportunityOptions.java

```diff
@@ -1,4 +1,29 @@
-package PACKAGE_NAME;
+public enum UpdateOpportunityOptions implements Menu {
 
-public enum UpdateOpportunityOptions {
+    NAME("n", "Update name");
+
+    private String command;
+    private String name;
+
+    UpdateOpportunityOptions(String command, String name) {
+        this.command = command;
+        this.name = name;
+    }
+
+    public static UpdateOpportunityOptions findChoice(String menuChoice) {
+        for (UpdateOpportunityOptions updateOpportunityOption : values()) {
```

Comment:
is there anyway to use some other method to do this? Feels like this pattern is happening a lot. Or some way to refactor

Reply:
Hi @ssunday, I have been looking into ways I can refactor this:
Enums aren't able to extend a class, so I was thinking of creating one large Enum with all possible menu choices, but adding a `menuType` to each constant indicating which type of `Menu` it belongs to e.g. starting menu.
Did you have any suggestions on how I can refactor this? 
Thanks!

Reply:
Oh, hmmmm. Maybe a combo of enum and a map?  Like the top level key is the menuType and then the values are the options.